pkg/virt-handler/isolation/validation.go

```diff
@@ -14,10 +17,12 @@ const (
 	QEMUIMGPath = "/usr/bin/qemu-img"
 )
 
-func GetImageInfo(imagePath string, context IsolationResult) (*containerdisk.DiskInfo, error) {
+func GetImageInfo(imagePath string, context IsolationResult, config *v1.DiskVerification) (*containerdisk.DiskInfo, error) {
+	memoryLimitMB := fmt.Sprintf("%d", config.MemoryLimit.ScaledValue(resource.Mega))
+
 	// #nosec g204 no risk to use MountNamespace()  argument as it returns a fixed string of "/proc/<pid>/ns/mnt"
 	out, err := virt_chroot.ExecChroot(
-		"--user", "qemu", "--memory", "1200", "--cpu", "10", "--mount", context.MountNamespace(), "exec", "--",
+		"--user", "qemu", "--memory", memoryLimitMB, "--cpu", "10", "--mount", context.MountNamespace(), "exec", "--",
```

Comment:
How about changing `--memory` to accept bytes? It would be great if we can stop the scaling around. We have a `Quantity` now on the API and we set the `rlimit` in bytes. This way we have no issues with scaling.

Reply:
Yes can do that, I left that as Mbytes mostly because I didn't want to change the API of the tool but it seems that the container disk validation is the only one that uses the parameter.